blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/FlowNodeWrapper.java

```diff
@@ -77,7 +81,11 @@ private static NodeType getNodeType(FlowNode node){
         throw new IllegalArgumentException(String.format("Unknown FlowNode %s, type: %s",node.getId(),node.getClass()));
     }
 
-    public @Nonnull NodeRunStatus getStatus(){
+    public @Nonnull NodeRunStatus getStatus() {
+        // Do not count block errors that are aborts
+        if (hasBlockError() && !isBlockErrorInterruptedWithAbort()) {
```

Comment:
@vivek Not checking the block error caused aborted input steps to be marked as failures. Oops. Will need your blessing.